include/fcl/narrowphase/detail/traversal/octree/collision/mesh_octree_collision_traversal_node.h

```diff
@@ -39,7 +39,7 @@
 #define FCL_TRAVERSAL_OCTREE_MESHOCTREECOLLISIONTRAVERSALNODE_H
 
 #include "fcl/config.h"
-#if not(FCL_HAVE_OCTOMAP)
+#ifndef FCL_HAVE_OCTOMAP
```

Comment:
`FCL_HAVE_OCTOMAP` will be always defined either `FCL_HAVE_OCTOMAP 0` or `FCL_HAVE_OCTOMAP 1` since we're using [`cmakedefine01`](https://github.com/flexible-collision-library/fcl/blob/master/include/fcl/config.h.in#L45) (see [CMake manual](https://cmake.org/cmake/help/v3.0/command/configure_file.html)).

Maybe `#if !(FCL_HAVE_OCTOMAP)` would work instead? (https://docs.microsoft.com/en-us/cpp/preprocessor/hash-if-hash-elif-hash-else-and-hash-endif-directives-c-cpp?view=vs-2019)